client/test/extension.test.ts

```diff
@@ -9,14 +9,38 @@ import * as assert from 'assert';
 // You can import and use all API from the 'vscode' module
 // as well as import your extension to test it
 import * as vscode from 'vscode';
+import * as path from 'path';
 import * as myExtension from '../src/extension';
+import { ExtensionContext, workspace, Uri } from 'vscode';
+import { error } from 'util';
+
 
 // Defines a Mocha test suite to group tests of similar kind together
 suite("Extension Tests", () => {
 
-	// Defines a Mocha unit test
-	test("Something 1", () => {
-		assert.equal(-1, [1, 2, 3].indexOf(5));
-		assert.equal(-1, [1, 2, 3].indexOf(0));
+
+	// test validation cto
+	test("activate", () => {
+		
+		console.log('workspace.rootPath=' + vscode.workspace.workspaceFolders);
+
+		let rootPath = path.dirname(__dirname);		
+		let uri = vscode.Uri.file(path.join(rootPath, '../test/data/test.cto'));
+
+		workspace.openTextDocument(uri).then((document) =>{
+
+			let text = document.getText();
+			console.log('text = ' + text);

```

Comment:
should not have console logging in the test output, as this will clutter the test file output